### PR TITLE
Update source to openquake inputs

### LINKED ISSUE #104: migrate source csv files from model-input repo to openquake-input
migrate source csv files from model-input repo to openquake-input

**add_data.sh**
- Changes between lines 169 - 239 ?
- All source inputs change from model-inputs to respective openquake-inputs except site_model exposure lines 175 -177

**- #Physical Exposure**
- [x] BldgExpRef_CA_master_v3p1.csv to BldgExpRef_CA_master_v3p2.csv
- Column change new column "OccType", update copy statement to reflect https://github.com/OpenDRR/openquake-inputs/issues/12)

**- #vs30**
- [x] site-vgrid_CA.csv
- Column change, update copy statement to reflect - sql  already updated in model-factory

**- #census**
- ~[ ] census-attributes-2016.csv to census-attributes.csv~
- [x] Filename is the same. No column changes. census-attributes-2016.csv

**- #sovi**
- [x] social-vulnerability-census_2021 to social-vulnerability-census.csv
- No column changes

- [x] social-vulnerability-index_2021.csv to social-vulnerability-index.csv
- No column changes

- ~[ ] sovi_thresholds_2021.csv to sovi_thresholds.csv~
- [x] Filename is the same. No column changes.  sovi_thresholds_2021.csv

**- #lut**
- No change for collapse_probability.csv

- [x]  Enhancement for future (https://github.com/OpenDRR/openquake-inputs/issues/11)

- [x] Retrofit costs table removed and not needed.  Remove lines 215 - 218, update in copy statement.

**- #GHSL**
- [x] Filename is the same. No column changes

- **#MH**
- [x] HTi_Sauid_2021.csv to HTi_Sauid.csv
- No column changes

- [x] HTi_threshold_2021.csv to HTi_threshold.csv
- No column changes

- [x] Hazard_threat_rating_threshold.csv is a new lut - update copy table statements to include.   New create tables in model factory.  Updated sql indicators in model factory.

- [x] Create_table_mh_rating_thresholds.sql
- Add extra run_psql Create_table_mh_rating_thresholds.sql line 234

- [x] Create_MH_risk_sauid_prioritization_prereq_tables.sql
- Replaces Create_MH_risk_building_ALL.sql, update line 235

- [x] Create_MH_risk_sauid_prioritization_Canada.sql
- Replaces Create_MH_risk_building_ALL.sql, update line 236

### PR DESCRIPTION
**Pull request by drotheram**
Migrate input data from `model-inputs` repo to `openquake-inputs` repo.

- Changes to source repos
- Changes to filenames
- Removal of retrofit costs

Changes defined by requirements outlined in:
https://github.com/OpenDRR/opendrr-api/issues/104

Closes #104

### CODE REVIEW

**Review by anthonyfok — APPROVED** [review]
Looks good to me!  Thank you Drew (and Will) for your great work!

File: python/add_data.sh [hunk 1]
```diff
@@ -508,42 +508,46 @@ import_sovi() {
...
+    exposure/general-building-stock/1.%20documentation/collapse_probability.csv
   RUN run_psql Create_collapse_probability_table.sql
 }
 
 import_retrofit_costs() {
   LOG "## Retrofit Costs"
-  RUN fetch_csv model-inputs \
-    exposure/general-building-stock/documentation/retrofit_costs.csv?ref=73d15ca7e48291ee98d8a8dd7fb49ae30548f34e
+  RUN fetch_csv openquake-inputs \
+    exposure/general-building-stock/1. documentation/retrofit_costs.csv
```
Comment (wkhchow) [inline python/add_data.sh:519]: should this be %20 to keep the same as above?

**Review by wkhchow — CHANGES_REQUESTED** [review]
few issues I noticed on the latest db dump
- [ ] mh.mh_rating_thresholds table is empty in the db.  Looks like csv is fetch in the stack, but not copied into the db  table in copyAncillaryTables.py.  Causes the result tables to be empty, and probably why the views are missing as well.
- [ ] vs30.vs30_can_site_model table is empty, I see in the copyAncillaryTables.py so not sure whats going on?  All the vs30_can_site_model_metrovan_*_exposure_xref tables are empty as well since vs30_can_site table is empty.

Besides that all looks good.

**Comment by drotheram** [comment]
Will add the mh_ratings_thresholds loading line. 

Traced the vs30_can_site_model table and found this error while loading with a Python system call to psql:

```
copy vs30.vs30_can_site_model (lon, lat, vs30, z1pt0, z2pt5) FROM '/usr/src/app/site-vgrid_CA.csv' WITH DELIMITER AS ',' CSV HEADER ;
ERROR:  unquoted newline found in data
HINT:  Use quoted CSV field to represent newline.
CONTEXT:  COPY vs30_can_site_model, line 2222069
256
```

**Comment by drotheram** [comment]
Fixed mh_rating_thresholds table copy command in model-factory

**Review by wkhchow — APPROVED** [review]

**Review by wkhchow — CHANGES_REQUESTED** [review]
awaiting db dump to verify again before approving.

**Review by jvanulde — APPROVED** [review]
Looks fine to me.

**Review by wkhchow — APPROVED** [review]

**Pull request merged**